_config.yml

```diff
@@ -1,13 +0,0 @@
-remote_theme: just-the-docs/just-the-docs
```

Comment:
Were planning to include the GitHub action update as part of this PR as well? I don't think I'm seeing it anywhere in the PR and would assume it would not be buildable by GitHub actions if we merged this as is.

Reply:
Sure!